homeassistant/components/sentry/sensor.py

```diff
@@ -0,0 +1,68 @@
+"""Support for Sentry SDK sensor."""
+import logging
+
+import sentry_sdk
+from sentry_sdk.integrations.logging import LoggingIntegration
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import Entity
+
+from .const import CONF_DSN, CONF_ENVIRONMENT
+
+
+class Sentry(Entity):
+    """Representation of the Sentry SDK."""
+
+    def __init__(self, hass: HomeAssistant, config: dict):
+        """Initialize Sentry sensor."""
+        self.hass = hass
+
+        self._name = config.get(CONF_NAME) or "sentry"
+        self._last_event_id = None
+
+        # https://docs.sentry.io/platforms/python/logging/
+        sentry_logging = LoggingIntegration(
+            level=logging.INFO,  # Capture info and above as breadcrumbs
+            event_level=logging.ERROR,  # Send errors as events
+        )
+
+        sentry_sdk.init(
+            dsn=config.get(CONF_DSN),
+            environment=config.get(CONF_ENVIRONMENT),
+            integrations=[sentry_logging],
+        )
+
+    @property
+    def name(self):
+        """Return the name."""
+        return self._name
+
+    @property
+    def state(self):
+        """Return the current state."""
+        return ""
```

Comment:
This feels weird. Why do we provide an entity to begin with ? And if it's about showing the last bug ID, shouldn't it make more sense to just make that the state ?

Reply:
Last ID isn't that important (and Sentry has an API to catch it, so might as well not store it elsewhere).